constraint/pkg/client/drivers/interface.go

```diff
@@ -45,6 +46,11 @@ type Driver interface {
 	// Dump outputs the entire state of compiled Templates, added Constraints, and
 	// cached data used for referential Constraints.
 	Dump(ctx context.Context) (string, error)
+
+	// TODO docs
+	EvaluationStats() string
+
+	StatsEntries() []*instrumentation.StatsEntry
```

Comment:
NOTE:

These two funcs server the same purpose: `exposing stats out of the driver`

I want us to pick one of the two and I am more partial to `StatsEntries` as it returns data in a more structured way and leaves it up to the consumers to define the format of emission (logging, otel, etc.)

That said, the previous implementation had even less structure to the stats and just exposed a string out of the driver.

Reply:
Given the above and chatting internally too, I really like the UX of sending over more structured data. So the `string` approach is going away in the latest commit.